Dockerfile

```diff
@@ -1,40 +1,73 @@
-FROM  ubuntu:xenial
+# Use node base image
+FROM node:latest
+ENV NODE_ENV=development
 
-# Add labels for metadata
+# Create a working app directory
+RUN mkdir -p /app
+WORKDIR /app
+
+# Copy the package.json file into app directory to install the npm packages
+COPY ./package.json /app
+
+# Install all the npm packages
+RUN npm install
```

Comment:
`yarn` not `npm`, creates a ghost `package-lock.json` otherwise. also ensure it's `--pure-lockfile`

Reply:
So can we use the flag ` yarn install --no-lockfile`?